source/elements/tags/Grid/Grid.css

```diff
@@ -0,0 +1,312 @@
+:root {
+  --grid--gap: 1.25rem;
+}
+
+.Grid {
+  @media (--small) {
+    display: flex;
+    flex: 0 1 auto;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 0 calc(var(--grid--gap) / -2);
+    padding: 0;
+
+    @supports (display: grid) {
+      display: grid;
+      grid-auto-flow: row;
+      grid-gap: var(--grid--gap);
+      grid-template-columns: repeat(2, 1fr);
+      margin: 0;
+    }
+  }
+
+  @media (--medium-small) {
+    @supports (display: grid) {
+      grid-template-columns: repeat(6, 1fr);
+    }
+  }
+
+  @media (--medium) {
+    margin: 0 auto;
+
+    @nest .wrapper & {
+      margin: 0 calc(var(--grid--gap) / -2);
+    }
+
+    @supports (display: grid) {
+      grid-template-columns: repeat(12, 1fr);
+
+      @nest .wrapper & {
+        margin: 0;
+      }
+    }
+  }
+
+  &--auto-fit {
+    @media (--small) {
+      @supports (display: grid) {
+        grid-template-columns: repeat(auto-fit, minmax(10rem, 1fr));
+        grid-template-rows: repeat(auto, 1fr);
+
+        & .Grid__item {
+          grid-column: auto;
+        }
+      }
+    }
+  }
+
+  &--auto-fill {
+    @media (--small) {
+      @supports (display: grid) {
+        grid-template-columns: repeat(auto-fill, minmax(10rem, 1fr));
+        grid-template-rows: repeat(auto, 1fr);
+
+        & .Grid__item {
+          grid-column: auto;
+        }
+      }
+    }
+  }
+
+  &__item {
+    margin: calc(var(--grid--gap) / 2) 0;
+
+    @media (--small) {
+      display: inline-block;
+      flex-basis: calc(50% - var(--grid--gap));
+      margin: calc(var(--grid--gap) / 2);
+      width: calc(50% - var(--grid--gap));
+
+      &--span-twelve {
+        flex-basis: 100%;
+        width: 100%;
+      }
+
+      @supports (display: grid) {
+        float: none;
+        margin: 0;
+        grid-column: auto / span 2;
+        width: auto;
+      }
+    }
+
+    @media (--medium-small) {
+      @supports (display: grid) {
+        grid-column: auto / span 3;
+
+        &--span-twelve {
+          grid-column: auto / span 6;
+        }
+      }
+    }
+
+    @media (--medium) {
```

Comment:
Consider adding:
https://www.npmjs.com/package/postcss-for
https://www.npmjs.com/package/postcss-at-rules-variables

To sum up all these column rules into a loop and also a grids variable file?

```css
	@for $i from 1 to var(--grid--column--count) {
		&--span-$i {
			flex-basis: calc(calc(calc($i/var(--grid--column--count)) * 100%) - var(--grid--gap));
			width: calc(calc(calc($i/var(--grid--column--count)) * 100%) - var(--grid--gap));
		}

		&--span-$i {
			grid-column: auto / span $i;
		}
	}
```